### DISCUSSION TITLE
Url rewrite with different locale domain

### DISCUSSION THREAD

**Discussion by fenes** [i18next/next-i18next#1800] [Q&A] [answered]
I want to rewrite some URLs for each locale with a different domain.
Here is i18N config;
```
i18n: {
    localePath: path.resolve("./public/locales"),
    defaultLocale: 'tr',
    locales: ['en', 'tr'],
    localeDetection: false,
    domains: [
      {
        domain: 'www.domain.com',
        defaultLocale: 'en',
      },
      {
        domain: 'www.domain.com.tr',
        defaultLocale: 'tr',
        http: true
      },
    ],
    rewrites: [
      {
        destination: '/tr/hakkimizda',
        source: '/aboutus',
        locale: false,
      },
      {
        source: '/tr/contactus',
        destination: '/tr/iletisim',
        locale: false,
      }
    ],
  }
```
For example,
I want to rewrite this URL => domain.com.tr/hakkimizda
here is the working url => domain.com.tr/aboutus
when I type domain.com.tr/hakkimizda , it has to be directed to aboutus(domain.com.tr/aboutus) page with TR locale

But I'm stuck into these rewrites block?
Any suggestions??
Thanks.

**Comment by isaachinman** [comment, marked as answer]
@fenes This functionality is unrelated to `next-i18next`, and should be discussed on the main NextJs repo.
